dist/main.js

```diff
@@ -0,0 +1,54 @@
+const apiManager = new APIManager()
+const renderer = new Render()
+const storageManager = new StorageManager()
+
+apiManager.getDreamPlayers(renderer.renderDreamTeam)
```

Comment:
You bulit this a little bit in an opposite way - the render should render whatever data the APIManager has